scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -2627,10 +2659,14 @@ def __init__(
         self.__controlled_warmer = controlled_warmer
 
         # give this an initial empty value
-        self.raw_logs = []
+        self.raw_logs = [] # type: List[DockerLog]
+        self.docker_logs = [] # type: List[DockerLog]
```

Comment:
I may have missed it but what was the purpose of having a separate docker_logs field?

Reply:
That's a good point. The only difference I can see is that raw_logs are already being monitored and docker_logs are those monitored and ready to be monitored.
Since it's a public field I didn't want to mess with it and kept the same logic, just used DockerLog instead of Dict make it more clear.
This monitor as a whole could be refactored to make it much less complex.